quivr/tables.py

```diff
@@ -668,9 +667,16 @@ def to_dataframe(self, flatten: bool = True) -> pd.DataFrame:
     def column(self, column_name: str) -> pa.ChunkedArray:
         """
         Returns the column with the given name as a raw pyarrow ChunkedArray.
+        Column can be a nested column, in which case, this function will recursively
+        search for the column through the nested subtables.
 
         :param column_name: The name of the column to return.
```

Comment:
Please add an example, ideally as a doctest, to clarify the syntax. Here's one that I haven't tried executing (`hatch run dev:doctest` will execute it):

```suggestion

        :param column_name: The name of the column to return.
        
        Examples:
            >>> import quivr as qv
            >>> import pyarrow.compute as pc
            >>> class MySubTable(qv.Table):
            ...     x = qv.Int64Column()
            ...     y = qv.Int64Column()
            >>> class MyWrapperTable(qv.Table):
            ...     child = MySubTable.as_column()
            >>> c = MySubTable.from_kwargs(x=[1, 2, 3], y=[4, 5, 6])
            >>> p = MyWrapperTable.from_kwargs(child=c)
            >>> column_x = p.column("c.x")
            >>> print(column_x.to_pylist())
            [1, 2, 3]
```